inscriptions/ISic003427.xml

```diff
@@ -185,11 +181,21 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#VM">
+                <ab>
+                    <lb n="1"/><supplied reason="lost">Δάματ</supplied>ρι καὶ Κόρᾳ
+                    <lb n="2"/><supplied reason="lost">ὁ</supplied> <supplied reason="lost">δεῖνα</supplied> <supplied reason="lost">Π</supplied>ρωτοφάνεος
```

Comment:
@simonastoyanova  presumably there is an EpiDoc way of encoding the equivalent of ο δεινα within a supplied element, rather than the ultimately misleading inclusion of Greek text for 'some name'...?

Reply:
In these cases we put a <note> in the supplied, with something like 'e.g' or 'κτλ' or however we want to express the editorial guesswork. We can still tag δεινα as a word and lemmatise, so it comes up in a word search but it will not show up in the names index.

Reply:
Since we actually want to flag two words - ὁ and δεινός, then we can put the <note> before their supplieds rather than inside. Then we don't have to do anything special about display, the words get picked up for a word search but ignored for the name index. We can also suppress them for the linguistic bit of the reseach.

Reply:
Final version of my brain-struggle here. The most logical approach seems to be to not lemmatise, since these are not actual words in the text but an editorial intervention. So we can lump them together in one `<supplied>` instead of two, and add a `<note>` inside that `<supplied>`, saying something like 'e.g.'

Reply:
I don't know... maybe, for the sake of consistency, even if we adopt Gentili's edition which uses this expedient, we could just insert a gap as elsewhere (as we do with numbers when we cannot reconstruct them)